4337/contracts/Safe4337Module.sol

```diff
@@ -20,28 +20,56 @@ import {ISafe} from "./interfaces/Safe.sol";
  *      - Replay protection is handled by the entry point.
  */
 contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandler {
+    /**
+     * @dev A structure used internally for computing the EIP-712 signing message for a Safe operation.
+     */
+    struct SafeOpFields {
```

Comment:
nitpick: I know style wise not optimal, but might be easier for varification if this is defined right after the `SAFE_OP_TYPEHASH` constant